auslib/test/blobs/test_apprelease.py

```diff
@@ -1644,20 +1644,20 @@ def setUp(self):
 {
     "name": "d1",
     "schema_version": 50,
-    "detailsUrl": "http://moo.com/%LOCALE%/cow/%VERSION%/",
+    "detailsUrl": "http://moo.com/%LOCALE%/cow/%VERSION%/%OS%",
     "displayVersion": "50.0"
 }
 """)
 
     def testDesupport(self):
-        updateQuery = {"locale": "<locale>", "version": "<version>"}
+        updateQuery = {"locale": "<locale>", "version": "<version>", "buildTarget": "<OS_Something>"}
```

Comment:
It would be better to use a real build target here. "Darwin_x86_64-gcc3-u-i386-x86_64" is probably the best one to use, since it is the most complex.